src/stylesString.js

```diff
@@ -2,7 +2,7 @@
 
 const stylesString = `
   background-color:      #fff;
--webkit-border-radius: 5px;
+  -webkit-border-radius: 5px;
```

Comment:
Source file changing is not right. You should pass test with original source file.